UltimateTeam.Toolkit/Models/WatchlistResponse.cs

```diff
@@ -2,13 +2,13 @@
 
 namespace UltimateTeam.Toolkit.Models
 {
-    public class WatchlistResponse
+    public class WatchlistResponse : IListResponse
```

Comment:
Why are 3 properties `virtual` here (only 2 were `virtual` in `AuctionResponse`)? Why should it be possible to override them?